lib/accesslists/hierarchy.go

```diff
@@ -541,7 +541,7 @@ func IsAccessListMember(
 
 	for _, member := range members {
 		// Is user an explicit member?
-		if member.Spec.MembershipKind != accesslist.MembershipKindList && member.GetName() == user.GetName() {
+		if member.Spec.MembershipKind != accesslist.MembershipKindList && member.Spec.Name == user.GetName() {
```

Comment:
Why `member.GetName() =` was replaced by `member.Spec.Name` ? 

Te `member.GetName()` is sorter + more readable + nil safe.

Reply:
Those were the only two cases of `.GetName`, was trying to keep things consistent. Can update the other cases to `GetName()` instead, though.

Reply:
Actually, I remember why – I could be wrong, but does `GetName()` on AccessListMember not return `Metadata.Name`, not `Spec.Name`?